docs/framework/install/net-framework-3-5-on-windows-8-plus.md

```diff
@@ -18,52 +18,25 @@ author: "mairaw"
-  
--   The .NET Framework 3.5 is also used by apps built for versions 2.0 and 3.0.  
-  
--   Installing a Windows language pack before installing the .NET Framework 3.5 may cause the .NET Framework 3.5 installation to fail. Install the .NET Framework 3.5 before installing any Windows language packs.  
-  
--   Windows CardSpace is not available with the .NET Framework 3.5 on [!INCLUDE[win8](../../../includes/win8-md.md)].  
-  
--   Because of complications around how the .NET Framework 3.5 must be installed, it is unfortunately not possible to provide a separate, standalone installer that can run independently of Windows Update. Again, if all other methods fail, you must resort to installation media as described earlier.  
-  
-<a name="OnDemand"></a>   
-## Install the .NET Framework 3.5 on Demand  
- If an app requires the .NET Framework 3.5, but doesn't find that version enabled on your computer, it displays the following message box, either during installation, or when you run the app for the first time. In the message box, choose **Install this feature** to enable the .NET Framework 3.5. This option requires an Internet connection.  
-  
- ![Dialog box for 3.5 install on Windows 8](../../../docs/framework/deployment/media/installdialog.png "installdialog")  
-  
-<a name="ControlPanel"></a>   
-## Enable the .NET Framework 3.5 in Control Panel  
- You can enable the .NET Framework 3.5 yourself through Control Panel. This option requires an Internet connection.  
-  
-1.  Press the Windows key ![Windows logo](../../../docs/framework/get-started/media/windowskeyboardlogo.png "Windowskeyboardlogo") on your keyboard, type Windows Features, and press Enter. This brings up the **Turn Windows features on or off** dialog box. Alternately, open Control Panel, click on the Programs items, and then click on “Turn Windows features on or off” under Programs and Features.  
-  
-2.  Select the **.NET Framework 3.5 (includes .NET 2.0 and 3.0)** check box, press OK, and reboot your computer if prompted.  
-  
- You do not need to select the child items for Windows Communication Foundation (WCF) HTTP activation unless you are a developer who requires WCF script and handler mapping functionality.  
-  
- The following video shows how to do this:  
-  
- ![Installing the .NET Framework on Windows 8 or 8.1](../../../docs/framework/get-started/media/clr-net35-win8.png "CLR_NET35_Win8")  
-  
-## See Also  
- [Installation Guide](../../../docs/framework/get-started/index.md)
\ No newline at end of file
+
+# Troubleshooting the installation of the .NET Framework 3.5 on Windows 8, Windows 8.1, and Windows 10
+
+During installation you may encounter error 0x800f0906, 0x800f0907, or 0x800f081f, in which case refer to [.NET Framework 3.5 installation error: 0x800f0906, 0x800f0907, or 0x800f081f](https://support.microsoft.com/en-us/kb/2734782). Note that these are possibly resolved by installing [security update 3005628](https://support.microsoft.com/kb/3005628).
```

Comment:
the link to the kb article 2734782 is not what i expected. i think we need to link to https://support.microsoft.com/en-us/help/2734782/.net-framework-3.5-installation-error-0x800f0906,-0x800f081f,-0x800f0907.